Classes/Client/ServiceBwClient.php

```diff
@@ -11,9 +11,11 @@
 
 namespace JWeiland\ServiceBw2\Client;
 
+use JWeiland\ServiceBw2\Client\Event\ModifyServiceBwResponseEvent;
 use JWeiland\ServiceBw2\Client\Helper\LocalizationHelper;
 use JWeiland\ServiceBw2\Client\Helper\TokenHelper;
 use JWeiland\ServiceBw2\Configuration\ExtConf;
+use Psr\EventDispatcher\EventDispatcherInterface;
```

Comment:
I would prefer using TYPO3s EventDispatcher as type instead of the Interface.

Reply:
TYPO3 itself uses the interface too. Do you see any advantage using TYPO3s EventDispatcher?

Reply:
PhpStorm will show you all available methods

Reply:
I would like to keep the Interface because TYPO3 never uses the EventDispatcher class for dependency injection. Also the documentation says that best practice is to use the Psr Interface (https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Hooks/EventDispatcher/Index.html#best-practices)